docs/run/get-backend-information.ipynb

```diff
@@ -101,241 +146,142 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Configuration\n",
+    "## Static backend information\n",
```

Comment:
Previously the page was split into "configuration" and "properties" sections. These methods don't exist on `BackendV2` so I've switched them for "static" vs "dynamic" information.